Sources/FaunaDB/Value.swift

```diff
@@ -331,45 +369,58 @@ extension BytesV: Equatable {
     }
 }
 
+/// Represents a query value in the FaunaDB query language.
+/// [Reference](https://fauna.com/documentation/queries#values-special_types).
+public struct QueryV: Value, AsJson {
+
+    internal let value: [String: Any]
```

Comment:
Since it has become a internal let, can't we make it a `JsonType` instead? That would let us free of the `wrap` function.